index.bs

```diff
@@ -1436,6 +1455,31 @@ To <dfn export>get user context</dfn> given |user context id|:
 
 </div>
 
+When a [=user context=] is [=set/remove|removed=] from the
+[=set of user contexts=], [=remove user context subscriptions=]:
```

Comment:
```suggestion
When a [=user context=] is [=set/remove|removed=] from the
[=set of user contexts=], [=remove user context subscriptions=].
```

But also, can we move this up into the paragraph starting "Implementations may [=set/remove=] any [=empty user context=], with exception of[…]" (I can't add that as a suggestion, sadly).